src/modm/platform/spi/stm32/spi_master_dma.hpp.in

```diff
@@ -83,8 +83,8 @@ private:
 	handleDmaTransmitComplete();
 
 	static inline bool dmaError { false };
-	static inline bool dmaTransmitComplete { false };
-	static inline bool dmaReceiveComplete { false };
+	static inline bool dmaTransmitComplete { true };
+	static inline bool dmaReceiveComplete { true };
```

Comment:
So the scenario is to send the SPI data with low frequency. It's not a real high-frequency poll, so in general the previous packet would have been sent before the next is ready, but I'd like to have a guard for this case as well.

Before I send a packet, I check, whether SPI is ready to send:

```c++
    if (SpiMaster::transfer(nullptr, nullptr, 0).getState() == modm::rf::Running) {
      return false;
    }
    SpiMaster::transfer(reinterpret_cast<uint8_t*>(data_.data()), nullptr, sizeof(data_));
    return true;
```

If nothing has been sent before this function is called, the `transfer` always return `Running` and the real send never occurs.

So there is a way to overcome this by sending some dummy at the initialization stage, but I thought it could be also nice to reorganize the code to handle the general case as well.

Also, for the initialisation values: `dmaTransmitComplete` generally marks, that no transfers are active, so i the `false` value could be valid here.

In general, I don't think this PR is important and can be simply deleted.